test/declaration/extension_type_comment.unit

```diff
@@ -0,0 +1,61 @@
+40 columns                              |
+>>> Inline block comments everywhere.
```

Comment:
Wow lol these tests are way more extensive than our typical comment tests.

Reply:
Yeah, Lasse wrote these and I just migrated them over.